assets/_locales/en/messages.json

```diff
@@ -1550,5 +1550,9 @@
     "settings_custom_position_below": {
         "message": "Below Information",
         "description": "Position of the quicklinks below the information section"
+    },
+    "settings_forceEn": {
```

Comment:
I am always a bit reluctant in translating language switches. In case you do not understand the language it is in you can still find your way. But in this case it is less of a concern.

Reply:
My opinion is also that it should be kept in english only

Reply:
I agree, if you don't know english enough to get what switch option tells you, you can't possibly understand the whole mess UI will turn after this switches. I'll revert this